src/validation/schemas.js

```diff
@@ -0,0 +1,26 @@
+import Joi from '@hapi/joi';
+
+/**
+ * user schema to be used for validating user input
+ */
+export const signUpSchema = {
+  user: Joi.object().keys({
```

Comment:
Why do you need to add user?  this can work 

`
export const signUpSchema = Joi.object().keys({
`

instead of this

`
 export const signUpSchema = {
  user: Joi.object().keys({
`